src/components/Editor/helpers/setZoomEvent.ts

```diff
@@ -18,20 +18,38 @@ export const setZoomEvent = (
     const event = e as unknown as WheelEvent
     if (!(event.ctrlKey || event.metaKey)) return
 
-    event.preventDefault()
-
-    let fontSize = parseFloat(editorElement.dom.style.fontSize)
-    if (isNaN(fontSize)) {
-      fontSize = storedFontSize
-    }
+    changeFontSize(event, editorElement, storedFontSize, event.deltaY < 0)
+  })
 
-    if (event.deltaY < 0) {
-      fontSize = Math.min(fontSize + 1, maxFontSize)
-    } else {
-      fontSize = Math.max(fontSize - 1, minFontSize)
-    }
+  element.addEventListener('keypress', (e) => {
```

Comment:
`keypress` only gets called for printable character and is deprecated. We should use `keydown` instead.
REF - https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event

Reply:
keypress -> keydown :+1: